panel/src/config/api.js

```diff
@@ -48,16 +49,13 @@ export default {
 					}
 
 					Vue.$api.requests.push(requestId);
-				},
-				onSuccess: () => {
-					clearInterval(Vue.$api.ping);
-					Vue.$api.ping = setInterval(Vue.$api.auth.user, 5 * 60 * 1000);
 				}
 			},
-			ping: null,
 			requests: []
 		});
 
-		Vue.$api.ping = setInterval(Vue.$api.auth.user, 5 * 60 * 1000);
+		// regularly ping API to keep session alive
+		// (if any use is authenticated)
```

Comment:
```suggestion
		// (if any user is authenticated)
```

Reply:
Actually I think we can remove this line entirely. The session needs to be kept alive even if no user is authenticated.